SUMMARY.md

```diff
@@ -49,7 +49,10 @@
   * [Rules Engine](foundation-extensions/mobile-core/rules-engine/README.md)
     * [Rules Engine technical details](foundation-extensions/mobile-core/rules-engine/rules-engine-details.md)
     * [Rules Engine consequence details](foundation-extensions/mobile-core/rules-engine/rules-engine-consequence-details.md)
-  * [Platform Services](foundation-extensions/mobile-core/platform-services.md)
+  * [Platform Services](foundation-extensions/mobile-core/platform-sercvies/README.md)
```

Comment:
misspelled `services` in all the links below.

Reply:
This seems to require file name fixes too, search for `sercvies`

Reply:
It's fixed in the PR, but not shown correctly on this page, you can find the updated doc on the "Files Changed" page.